File: 30 days of code( python)/Day 21.cpp
/**
*    Name: printArray
*    Print each element of the generic vector on a new line. Do not return anything.
*    @param A generic vector
**/

// Write your code here
template <class T>
void printArray(vector<T> my_vector)
{
    typename vector<T>::iterator it;
    for (it = my_vector.begin(); it < my_vector.end(); it++)
        {
        cout << *it << endl;
    }
}

File: 30daysOFcode_Java/day21.java
import java.util.*;

class Printer <T> {

    /**
    *    Method Name: printArray
    *    Print each element of the generic array on a new line. Do not return anything.
    *    @param A generic array
    **/
    
    // Write your code here

    public static <E> void printArray(E[] a)
    {
        for(E elements: a)
        {
            System.out.println(elements);
        }
    }
    


}
